platform/fabricx/core/finality/provider.go

```diff
@@ -41,14 +41,16 @@ func NewListenerManagerProvider(fnsp *fabric.NetworkServiceProvider, configProvi
 		fnsp:           fnsp,
 		configProvider: configProvider,
 		managers:       make(map[string]ListenerManager),
+		initOnce:       sync.Once{},
```

Comment:
this is not needed ... the zero-value of `sync.Once` is valid. Note that is similar to a `sync.Mutex`.